lighthouse-cli/test/smokehouse/test-definitions/offline-local/offline-expectations.js

```diff
@@ -57,9 +57,7 @@ module.exports = [
           scoreDisplayMode: 'notApplicable',
         },
         'installable-manifest': {
-          score: 0,
```

Comment:
this seems like a surprising change?

Reply:
Yeah this is a good point.  good catch

installableErrors doesnt emit unless there's a manifest.. but clearly you're not installable w/o a manifest.. so...

this does suggest we still need to surface the manifestValues.parseFailureReason as one of the "errors"

Reply:
good call 👍